src/bindings/c/include/openvino/c/ov_tensor.h

```diff
@@ -57,6 +57,17 @@ ov_tensor_create(const ov_element_type_e type, const ov_shape_t shape, ov_tensor
 OPENVINO_C_API(ov_status_e)
 ov_tensor_set_shape(ov_tensor_t* tensor, const ov_shape_t shape);
 
+/**
+ * @brief Constructs a new tensor using a string array.
+ * @ingroup ov_tensor_c_api
+ * @param string_array An array of strings
+ * @param array_size The size of the string array
+ * @param tensor A point to ov_tensor_t
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_tensor_create_from_string_array(const char** string_array, size_t array_size, ov_tensor_t** tensor);
```

Comment:
I think we also need to have ` const ov_shape_t shape` aurgument. String tensor can be of any shape as numeric one.

Reply:
Maybe change `array_size` to `shape` where shape size is number of strings in `string_array`